js/server/assets/swagger/index.html

```diff
@@ -86,8 +86,8 @@
         ],
         layout: "StandaloneLayout",
         //#region ArangoDB-specific changes
-        // Expand the API documentation by default.
-        docExpansion: "list",
+        validatorUrl: null,
+        docExpansion: "none",
```

Comment:
See the other PRs. I'd prefer keeping this as `"list"` so we don't have to backport #11579 to every version.

Reply:
Restored to "list"